lib/vsc/utils/nagios.py

```diff
@@ -141,7 +139,7 @@ def exit_from_errorcode(errorcode, msg, error_map=None):
     try:
         NAGIOS_EXIT_MAP[e_map[errorcode]](msg)
     except (IndexError, KeyError):
-        unknown_exit(msg)
+        unknown_exit(msg + " (errorcode not found in {0}".format(e_map))
```

Comment:
put the errocode stuff before the msg, because if it gets too long it will get cut off later in vsc.utils.nagios to protect the database